egs/wsj/s5/steps/nnet3/tdnn/make_configs.py

```diff
@@ -194,6 +221,23 @@ def PrintConfig(file_name, config_lines):
     f.write("\n".join(config_lines['component-nodes']))
     f.close()
 
+def ParseCnnString(cnn_string):
```

Comment:
Yes, it could be cnn_param_string_list.

On Thu, Mar 3, 2016 at 4:48 AM, tomkocse <notifications@github.com> wrote:

> In egs/wsj/s5/steps/nnet3/tdnn/make_configs.py
> <https://github.com/kaldi-asr/kaldi/pull/530#discussion_r54856766>:
>
> > @@ -194,6 +221,23 @@ def PrintConfig(file_name, config_lines):
> >      f.write("\n".join(config_lines['component-nodes']))
> >      f.close()
> >
> > +def ParseCnnString(cnn_string):
>
> Do you mean not using the name "cnn_string" ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/530/files#r54856766>.
>